plugins/tiddlywiki/codemirror/engine.js

```diff
@@ -130,6 +130,58 @@ function CodeMirrorEngine(options) {
 		if(!self.widget.isFileDropEnabled) {
 			event.stopPropagation(); // Otherwise TW's dropzone widget sees the drop event
 		}
+		// Detect if Chrome has added a pseudo File object to the dataTransfer
+		if(!$tw.utils.dragEventContainsFiles(event) && event.dataTransfer.files.length) {
+			//Make codemirror ignore the event as we will handle the drop ourselves
+			event.codemirrorIgnore = true;
+			event.preventDefault();
+
+			// from https://github.com/codemirror/CodeMirror/blob/master/src/measurement/position_measurement.js#L673
+			function posFromMouse(cm, e, liberal, forRect) {
```

Comment:
@Jermolene regarding the `posFromMouse()` inline method copied verbatim from CodeMirror, should it be kept as is or brought in line with our coding standards? It uses `let` and `destructuring assignments`, and semi-colons are nowhere to be seen.

Reply:
I guess if this code is already in the CodeMirror plugin for v5.2.0 it won't cause any further compatibility problems, so I think we should leave as is.